src/styles/blocks/card.css

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card__img {
+  display: block;
```

Comment:
isn't it the default style for img?

Reply:
[i don't think so.](https://stackoverflow.com/questions/2402761/is-img-element-block-level-or-inline-level) `display: block;` helps me set horizontal align. Am i right?